fe/fe-core/src/main/java/com/starrocks/StarRocksFE.java

```diff
@@ -137,45 +132,6 @@ public static void start(String starRocksDir, String pidDir, String[] args) {
         }
     }
 
-    // NOTE: To avoid dead lock
-    //      1. never call System.exit in shutdownHook
-    //      2. shutdownHook cannot have lock conflict with the function calling System.exit
-    private static void addShutdownHook() {
-        Runtime.getRuntime().addShutdownHook(new Thread(() -> {
```

Comment:
Must add a more concise commit message about the problem and solution.
Don't make me guess what's the problem is.

Reply:
The problem is described in issue.